apps/web/src/app/(dashboard)/admin/stats/page.tsx

```diff
@@ -70,6 +105,20 @@ export default async function AdminStatsPage() {
           </div>
         </div>
       </div>
+
+      <div className="mt-16">
+        <h3 className="text-3xl font-semibold">User Charts</h3>
+
+        <UserWithDocumentChart
+          data={MONTHLY_USERS_SIGNED}
+          className="col-span-12 mb-8 mt-4 lg:col-span-6"
+        />
+
+        <UserWithDocumentCummulativeChart
+          data={MONTHLY_USERS_SIGNED}
+          className="col-span-12 mb-8 mt-4 lg:col-span-6"
```

Comment:
Do the col-spans do anything in this case?

Reply:
i guess not, i'll remove it